src/index.ts

```diff
@@ -16,13 +17,14 @@ import {NgbTooltipModule} from './tooltip';
 import {NgbTypeaheadModule} from './typeahead';
 
 export {NgbPanelChangeEvent} from './accordion';
+export {NgbModal} from './modal';
```

Comment:
They should, I thought I've exported them. This is still WIP so thx for catching this one!